src/js/other-websites/add_input_icon.js

```diff
@@ -200,6 +220,16 @@ async function addRelayIconToInput(emailInput) {
   sendInPageEvent("input-icon-injected", "input-icon");
 }
 
+function updateIframeHeight(height) {
+  const relayInPageMenuIframe = document.querySelector(".fx-relay-menu-iframe iframe");
+
+  // BUG: Console error in background.js being called. The code below solves it. 
+  if (relayInPageMenuIframe) {
+    relayInPageMenuIframe.height = height;
+  }
```

Comment:
Note: Similar to another comment above - Duplicate calls are happening, but this stops the noise.